src/utility/env.go

```diff
@@ -6,9 +6,9 @@ import (
 )
 
 func GetEnvAsserted(envVarName string) string {
```

Comment:
`envVarName`?  Either use `name` or something shorter, or go full microsoft hungarian notation and name it `sNameOfTheEnvironmentVariableThatWeAreTryingToRetrieveFromTheEnvironmentThatTherProcessIsRunningIn`.

From package "os":
```
func Getenv(key string) string
```

Also, there's this `Getenv` and there's the wonderfully named `godotenv` package (which doesn't parse `go.env`, of course, but it's ported from Roby so of course it's idiotic).  I didn't look into the interaction between the two, but I hope you know what you're doing.